.azure-pipelines/windows/basic-tests.yml

```diff
@@ -0,0 +1,21 @@
+#
+# Azure Pipelines configuration for building and testing react on Windows.
+#
+
+steps:
+  - script: yarn build
+    displayName: 'Build'
+
+  - script: set JEST_JUNIT_OUTPUT=.azure-pipelines/test-results/test-basic-windows.xml&&yarn test
+    displayName: 'Run tests'
+
+  - script: yarn prettier
```

Comment:
Any idea why prettier is invoked differently in Windows vs. Linux? In Linux, the step has this:

```
node ./scripts/prettier/index
```

Reply:
I don't think it's intentional; more a side effect of having developed the AppVeyor and Circle processes separately. `yarn prettier` runs `node ./scripts/prettier/index write-changed` which is probably only appropriate locally. So I would suggest either changing the line you highlighted to `node ./scripts/prettier/index` or adding another script to the package. I'm leaning adding `prettier-ci` to the package.json...what do you think?

Reply:
If anything...if we add `prettier-ci` to the package, we can add the `git fetch origin master:master` to that script.